src/verifier/validateOptions.spec.ts

```diff
@@ -47,6 +47,18 @@ describe('Verifier argument validator', () => {
         });
       });
     });
+
+    context('when given an unknown array argument', () => {
+      it('should return a Verifier object', () => {
+        expectSuccessWith({
+          madeupArg: [''],
```

Comment:
would  js treat `['']` as different to `[]` (because js)

and would that make a diff for this logic `(rules || []).forEach((rule) => {`

Reply:
Nope 👍 

`rules` is from the rules object above - which is only accessed using the key. The value is only relevant for entering in the `Array.isArray` branch, which was missing the undefined check.

You can see the same `(rules || [])` pattern on the other branch - I'm not wild about having inline defaults, but I didn't want to change the code too much just to match personal preferences.